content/gettingstarted/index.adoc

```diff
@@ -16,129 +16,132 @@ toc::[]
 :icons: font
 :imagesdir: /images/gettingstarted/
 
-icon:bullhorn[size=2x]{nbsp} Kiali works with Istio versions *1.0.5+* and *1.1*.
+icon:bullhorn[size=1x]{nbsp} Kiali works with Istio versions *1.0.5+* and *1.1*.
  + 
 Kiali will support the latest Istio version (1.1 at the moment) and the development
 is closely following the development of Istio.
 Older Istio versions like 1.0.x may continue to work, but will no longer be actively supported.
 
-== Getting started on OpenShift
+icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenShift CLI command `oc` is used to interact with the cluster environment. If you are on Kubernetes, simply replace `oc` with `kubectl` unless otherwise noted.
 
-To deploy Kiali to your OpenShift cluster you can do the following.
+== Installing Kiali
 
 === Preparation
```

Comment:
Would "Prerequisites" be a better heading here?

Reply:
"Prerequisites" makes it sound like I need some things already installed or some dependencies deployed. This really is how to prepare your OpenShift or Kubernetes environment in order to be able to run Kiali

Reply:
Still things you need before you can run Kiali, but it's fine either way.  Not a hill I want to die on.  ;-)